staticjinja/staticjinja.py

```diff
@@ -59,18 +81,20 @@ def __init__(self,
                  outpath,
                  encoding,
                  logger,
-                 contexts=None,
+                 contexts=[],
```

Comment:
I forgot this trap (I'm sure I read about it at some point). It doesn't seem very likely this would be a problem here since this arguments is not meant to be changed at all. But you're right to stick a strict policy here.